src/main/java/pl/tomaszdziurko/jvm_bloggers/view/admin/blogs/BlogsPage.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en" xmlns="http://www.w3.org/1999/xhtml" xmlns:wicket="http://wicket.apache.org/dtds.data/wicket-xhtml1.5-strict.dtd">
+
+<head></head>
+
+<body>
+
+<wicket:extend>
+
+    <div id="page-wrapper">
+        <div class="row">
+            <div class="col-lg-12">
+                <h1 class="page-header">Blogs Managment Panel</h1>
+            </div>
+            <!-- /.col-lg-12 -->
+        </div>
+
+        <!-- /.row -->
+        <div class="row">
+            <div class="col-lg-12">
+                <div class="panel panel-primary">
+                    <div class="panel-heading">
+                        <i class="fa fa-rss fa-fw"></i> Latest blogs
```

Comment:
Not `latest`, we do not sort them in such order. Maybe something like `Blogs on the list` would be enough?